src/main/java/org/glassfish/spec/maven/SetPropertiesMojo.java

```diff
@@ -44,6 +44,12 @@
     @Parameter(defaultValue = "${project}", required = true, readonly = true)
     private MavenProject project;
 
+    /**
+     * Mode. Allowed values are "javaee", "jakarta"
+     */
+    @Parameter(property = "specMode", defaultValue = "javaee")
```

Comment:
Again, consider changing the default here as well.

Reply:
Done